src/coreclr/tools/Common/Compiler/DisplayNameHelpers.cs

```diff
@@ -11,6 +12,21 @@ namespace ILCompiler
 {
     internal static class DisplayNameHelpers
     {
+        public static string GetDisplayName(this TypeSystemEntity entity)
+        {
+            return entity switch
+            {
+                MethodDesc method => method.GetDisplayName(),
+                FieldDesc field => field.GetDisplayName(),
+                TypeDesc type => type.GetDisplayName(),
+#if !READYTORUN
+                PropertyPseudoDesc property => property.GetDisplayName(),
+                EventPseudoDesc @event => @event.GetDisplayName(),
```

Comment:
Where is the overload that this ends up calling? We only have overload for `PropertyPseudoDesc` in this file.

Reply:
Good catch. That's a bug. I'll fix it on NativeAOT side.